libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4090,6 +4091,13 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
                     break;
 #endif
```

Comment:
Need to change this #endif to before the break (previous code would fall-through when KDECAN isn't included, but with ToshibaCAN below that can't happen).

Reply:
great, thanks, fixed in follow-up commit